sklearn_pandas/cross_validation.py

```diff
@@ -1,13 +1,24 @@
+import warnings
 from sklearn import cross_validation
 from sklearn import grid_search
 
+deprecation_msg = '''
```

Comment:
I would upcase this variable name to denote that it's a constant, but that's just a style preference.